addons/portal/controllers/mail.py

```diff
@@ -115,6 +115,8 @@ def _portal_post_has_content(self, res_model, res_id, message, attachment_ids=No
     @http.route('/mail/avatar/mail.message/<int:res_id>/author_avatar/<int:width>x<int:height>', type='http', auth='public')
     def portal_avatar(self, res_id=None, height=50, width=50, access_token=None, _hash=None, pid=None):
         """ Get the avatar image in the chatter of the portal """
+        # TODO adapt the code in master as the route is never called without an
+        # "access_token" or without a "_hash" and a "pid".
```

Comment:
I'd keep the code as is, one line (already coded) to have it support that case instead of having an unpredictable behavior